android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -2384,4 +2401,53 @@ public Map<String, Object> getConstants()
 
         return constants;
     }
+
+    // Version String
```

Comment:
```suggestion
    //
    // Version Utils
    //
```
If we want to mark sections such that it shows up in the IDE (under "structure") then you can use

```
//region Version Utils
...
//endregion
```